client/src/pages/admin/admin-page.jsx

```diff
@@ -1,9 +1,62 @@
+import { useEffect, useState } from "react";
+import { getAllBikes } from "../../api/index";
+
 const AdminPage = () => {
-  return (
-    <>
-      <div className="main-container">Admin page</div>
-    </>
-  );
+   const [bikes, setBikes] = useState([]);
+   useEffect(() => {
+      getAllBikes()
+         .then((value) => value.data)
+         .then((val) => {
+            setBikes(val.data);
+         })
+         .finally();
+   }, [bikes]);
+
+   function reservationStatusLogic(bikeReservationStatus) {
```

Comment:
`const reservationStatusLogic = (bikeReservationStatus) => {`
please change it to ES6 
line 23 as well

Reply:
done